xdsl/interactive/app.tcss

```diff
@@ -1,5 +1,11 @@
 
-# Vertical(TextArea)
+# Horizontal(Button)
+#clear_input{
+    height: 3;
```

Comment:
is the height necessary? I'm not sure we need either of these css rules, can you please check if it looks any different without this?

Reply:
In the future, I'd suggest keeping these UI tweaks in separate PR's or add screenshots with the before/after maybe as @superlopuh has also requested.